src/pages/login/login.module.ts

```diff
@@ -1,6 +1,5 @@
 import { NgModule } from '@angular/core';
 import { IonicPageModule } from 'ionic-angular';
```

Comment:
@bjsawyer linter complains about line 2 for me as well if you want to remove this unused import as part of this PR as well

Reply:
Ah good catch. I think once we build out the Login page we'll end up needing that import so we can leave it.